tools/malwarejail/env/wscript.js

```diff
@@ -2201,3 +2183,62 @@ ga = function () {
 XPathResult = function () {
     return XPathResult;
 }
+
+// These constructors override the original class via aliasing.
+Blob = function() {
+    const _id = ++_object_id;
```

Comment:
Looking at _object_id, could you double-check the usage of it in the _addElementById function of Document in browser.js? I believe it doesn't increment it like every other places.

Reply:
Looks like Element is increasing via Node, so that way it's using the same id for both "id" and "_id".